pkg/xray/resource_xray_security_policy.go

```diff
@@ -67,8 +68,9 @@ func resourceXraySecurityPolicyV2() *schema.Resource {
 							Elem: &schema.Resource{
 								Schema: map[string]*schema.Schema{
 									"min_severity": {
-										Type:     schema.TypeString,
-										Optional: true,
+										Type:             schema.TypeString,
+										Optional:         true,
+										ValidateDiagFunc: validation.ToDiagFunc(validation.StringInSlice([]string{"All Severities", "Critical", "High", "Medium", "Low"}, true)),
```

Comment:
Is the string "all severities" legit?

Reply:
Yes, example is [here](https://www.jfrog.com/confluence/display/JFROG/Xray+REST+API#XrayRESTAPI-GetPolicy)
Same in the UI.